client/my-sites/marketing/buttons/preview-buttons.jsx

```diff
@@ -178,6 +178,7 @@ class SharingButtonsPreviewButtons extends Component {
 				width="100%"
 				frameBorder="0"
 				className="official-preview"
+				role="presentation"
```

Comment:
What do you think about adding this new property to `propTypes` as a `string` type to the `ResizableIframe` component?

Reply:
Let's do it 🚀